src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -20,6 +20,59 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE
     /// </summary>
     internal sealed class PEFieldSymbol : FieldSymbol
     {
+        private struct PackedFlags
+        {
+            // Layout:
+            // |..............................|vvvvv|
+            //
+            // f = FlowAnalysisAnnotations. 5 bits (4 value bits + 1 completion bit).
+
+            private const int HasDisallowNullAttribute = 0x1 << 0;
+            private const int HasAllowNullAttribute = 0x1 << 1;
+            private const int HasMaybeNullAttribute = 0x1 << 2;
+            private const int HasNotNullAttribute = 0x1 << 3;
+            private const int FlowAnalysisAnnotationsCompletionBit = 0x1 << 4;
+
+            private int _bits;
+
+#if DEBUG
+            static PackedFlags()
+            {
+                // Verify a few things about the values we combine into flags.  This way, if they ever
+                // change, this will get hit and you will know you have to update this type as well.
```

Comment:
Consider simplifying or eliminating this comment. In [https://github.com/dotnet/roslyn/pull/36152](https://github.com/dotnet/roslyn/pull/36152), all similar comments have been changed to "Verify masks are sufficient for values."